pkg/controller/nodenetworkconfigurationpolicy/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -165,33 +166,39 @@ func (r *ReconcileNodeNetworkConfigurationPolicy) Reconcile(request reconcile.Re
 		log.Error(err, "Error initializing enactment")
 	}
 
-	conditionsManager := conditions.NewManager(r.client, nodeName, *instance)
+	enactmentConditionsNotifier := enactmentconditions.NewNotifier(r.client, nodeName, instance)
+
+	// Policy conditions will be updated at the end so updating it
+	// does not impact at applyting state, it will increase just
+	// reconcile time.
+	defer conditions.UpdatePolicyConditions(r.client, instance)
```

Comment:
Originally I started with `enactmentconditions` and `policyconditions` but since we are already under `nodenetworkconfigurationpolicy` package I did the current approach, but I don't like it either, gong back to original.